tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "allowJs": true,
+    "allowJs": false,
```

Comment:
This was leading to superr slow type-checking time. No need to process all the js stuff, just stick to typescript code.